docs/reference/sql/functions/aggs.asciidoc

```diff
@@ -113,6 +113,185 @@ Returns the total number of _distinct non-null_ values in input values.
 include-tagged::{sql-specs}/docs.csv-spec[aggCountDistinct]
 --------------------------------------------------
 
+[[sql-functions-aggs-first]]
+===== `FIRST/FIRST_VALUE`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+FIRST(field_name<1>)
+FIRST_VALUE(field_name<1>)
+
+FIRST(field_name<1>, ordering_field_name<2>)
+FIRST_VALUE(field_name<1>, ordering_field_name<2>)
```

Comment:
Square brackets for optional fields, please.

Reply:
But now we list both with 1 arg and 2args, do you think is still necessary?

Reply:
If we do it like this (all versions listed - with/without the optional parameter), our documentation will be inconsistent: https://github.com/elastic/elasticsearch/blob/master/docs/reference/sql/functions/string.asciidoc#locate. If I would have to choose between the two approaches, I would go with `[ ]`.

Reply:
In my opinion you either have one signature with the 2nd arg in square brackets or you list the 2 variants.

Reply:
I opt for the signature with the second arguments in square brackets.